test/unittests/commonMocks.h

```diff
@@ -36,133 +40,163 @@
 
 /* ****************************************************************************
 *
-* USING 
+* USING
 */
 using ::testing::_;
```

Comment:
Keeping this here doesn't follow [M12 rule](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/contribution_guidelines.md#m12-using-namespace-in-header-files). However, I think we should make an exceptio here. Otherwise I guess that:

```
ON_CALL(*this, count(_, _, _, _, _))
```
will become in

```
ON_CALL(*this, count(::testing::_,::testing::_,::testing::_,::testing::_,::testing::_))
```

An that's crazy.

However, maybe the using statement could be moved inside the `DBClientConnectionMock` class in order to have a narrower scope?